packages/perseus-core/src/types.ts

```diff
@@ -1,19 +1,9 @@
 // Types that can be shared between Perseus packages
 // ideally without causing circular dependencies
 
-// TODO: this should be typed
-type State = any;
-
-// Interfact currently only implemented by
-// ServerItemRenderer and used by KeypadContext
-// to pass around a renderer reference
-export interface RendererInterface {
-    getSerializedState(): State;
-    restoreSerializedState(state: State, callback?: () => void): void;
-    scoreInput(): KEScore;
+// Used by KeypadContext to pass around a renderer reference
+export interface KeypadContextRendererInterface {
     blur(): void;
-    focus(): boolean | null | undefined;
-    props: any;
```

Comment:
So it _only_ needs `blur`?  Nice that we can trim it down that far... and thanks for clarifying the name!

Reply:
I hope so 🤞 I couldn't find where anything else was needed in either Perseus or Webapp, although there's some misdirection in Webapp that might prove me wrong.